presets/mainnet/merge.yaml

```diff
@@ -1,3 +1,16 @@
 # Mainnet preset - The Merge
 
-# No presets here.
+# Execution
+# ---------------------------------------------------------------
+# 2**20 (= 1,048,576)
+MAX_BYTES_PER_TRANSACTION: 1048576
+# 2**14 (= 16,384)
+MAX_TRANSACTIONS_PER_PAYLOAD: 16384
+# 2**8 (= 256)
+BYTES_PER_LOGS_BLOOM: 256
+# 2**10 (= 1,024)
+GAS_LIMIT_DENOMINATOR: 1024
+# 5,000
+MIN_GAS_LIMIT: 5000
+# 2**5 (= 32)
+MAX_EXTRA_DATA_BYTES: 32
```

Comment:
Shouldn't `BYTES_PER_LOGS_BLOOM` and `MAX_EXTRA_DATA_BYTES` be constant values disregarding the network, i.e. out of the scope of *any* configuration, even the preset one?

Reply:
This is fine. Presets offer compile-time configuration for clients that want to optimize usage of these and/or use static typing. They are not configurable per network, unless the testnet warrants a customized preset and clients compile with that. Flexible enough if we ever need a network that changes them, but not part of the default runtime config. See config/preset docs for more info.